components/rating/stories/rating.stories.js

```diff
@@ -1,23 +1,24 @@
 import { disableDefaultModes } from "@spectrum-css/preview/modes";
-import { isDisabled, isEmphasized, isFocused, isReadOnly } from "@spectrum-css/preview/types";
+import { isDisabled, isEmphasized, isKeyboardFocused, isReadOnly } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
 import { RatingGroup } from "./rating.test.js";
 import { Template } from "./template.js";
 
 /**
  * The rating component is used to display or collect a user's rating of an item as represented by a number of stars.
- * 
+ *
  * ### Usage notes
```

Comment:
Should this actually be an h2? Most of the other components that have "Usage notes" have it as an h2.

Reply:
Just updated this! ✨